en/config/actuators.md

```diff
@@ -1,139 +1,484 @@
+:::
+
+Sliders can be used to verify the following:
+
+1. Actuators (Motors, Control surfaces, etc.) are assigned to the expected output.
+1. Motors don't spin when at the `disarmed` PWM output value
+1. Motors barely spin at the `minimum` PWM output value.
+1. Motors give **positive thrust** in the expected direction
+1. Control Surfaces are in the correct idle position for `disarmed` output value
+1. Control Surfaces move in the direction as defined in the [Control Surface Convention](#control-surface-direction-conceptual)
+1. Motor Tilt Servos are in the correct idle position for `disarmed` output value
+1. Motor Tilt Servos move in the direction as defined in the [Tilt Servo Convention](#motor-tilt-servos-conceptual)
+
+
+## Output Assignment and Configuration
+
+In the _Actuator Outputs_ section on the right panel, you can assign and configure outputs, optionally using the [Actuator Testing](#actuator-testing) sliders:
+- MC vehicles that have connected motors to PWM outputs can use the [Identify & Assign Motors](#multicopter-pwm-identify-assign-motors) button to perform motor assignment "semi-automatically".
+- Motor output assignment can be done/checked using sliders (see [Output Assignment (Manual)](#output-assignment-manual)).
+- Actuator output assignment can be done/checked using sliders (see [Output Assignment (Manual)](#output-assignment-manual)). 
+- Disarmed, minimum, and maximum settings, for all outputs can also be also determined using sliders.
+  This is shown as part of [Motor Assignment & Configuration](#motor-assignment-configuration), [Control Surface Setup](#control-surface-setup), [Tilt servo setup](#tilt-servo-setup)
+
+### Multicopter PWM: Motor Assignment
+
+You can use the **Identify & Assign Motors** button to assign motors to PWM outputs using a semi-automated process.
+
+:::note
+This is the easiest way to assign motors, but is currently only supported for motors on **multicopter vehicles** that are connected to PWM outputs (UAVCAN outputs and other frame types do not support this feature).
+On other frames you can follow the instructions in [Output Assignment (Manual)](#output-assignment-manual).
+:::
+
+:::warning
+Remove the propellers from motors before assigning outputs or any testing.
+:::
+
+![Identify motor button](../../assets/config/actuators/identify_motors_button.png)
+
+When you click the button, QGC sends a command to a motor, causing it to spin.
+To assign that motor to an output you simply select the corresponding motor displayed in the screen.
+QGC will then spin the next motor for you to assign, and so on.
+
+Instructions:
+1. Setup the motor geometry to match the motors on your frame.
+1. Select the PWM tab where you want to assign the motors.
+1. Click the **Identify & Assign Motors** button.
+1. One motor will start spinning (click **Spin Motor Again** if it stops spinning too quickly to note.)
+
+   Click on the corresponding motor in the geometry section.
```

Comment:
Changing this back. Click and select are synonyms for a button, but click does not work on a touch screen.

```suggestion
   Select the corresponding motor in the geometry section.
```

Reply:
Haven't thought about that!